pallets/lbp/src/lib.rs

```diff
@@ -1111,6 +1127,18 @@ impl<T: Config> AMM<T::AccountId, AssetId, AssetPair, BalanceOf<T>> for Pallet<T
 			fee_amount: transfer.fee.1,
 		});
 
+		pallet_trade_event::Pallet::<T>::deposit_trade_event(
+			transfer.origin.clone(),
```

Comment:
I am thinking to add second account because of the Intents... We might have 2 accounts trading directly. cc @enthusiastmartin wdyt?

Reply:
Also easier to track when if have multiple pools with same assets

Reply:
not sure if the second account is good idea.. it is just confusing to me - what does the second account mean really ?  then you would need to amounts too. 

if you want to capture the matched trade in ICE - why not just fire 2 trade events for each account. The source of this wont be any AMM, but something else - eg  ICED.